fastai/data/core.py

```diff
@@ -217,8 +217,20 @@ def n_subsets(self): return len(self.splits)
     def _new(self, items, **kwargs): return super()._new(items, splits=self.splits, **kwargs)
     def subset(self): raise NotImplemented
 
-    def dataloaders(self, bs=64, shuffle_train=None, shuffle=True, val_shuffle=False,n=None, path='.', dl_type=None, dl_kwargs=None,
-                    device=None,drop_last=None,val_bs=None, **kwargs):
+    def dataloaders(self,
+        bs:int=64, # Size of batch
+        shuffle_train:bool=None, # Whether to shuffle training `DataLoader`
+        shuffle:bool=True, # Whether to shuffle training `DataLoader`
+        val_shuffle:bool=False, # Whether to shuffle validation `DataLoader`
+        n:int=None, # Size of `Datasets` used to create `DataLoader`
+        path:(str, Path)='.', # Path to put `DataLoaders`
+        dl_type=None, # Type of `DataLoader`
+        dl_kwargs=None, # Keyword arguments for individual `DataLoader`s
+        device=None, # Device to put `DataLoaders`
+        drop_last:bool=None, # Whether to drop last incomplete batch
+        val_bs:int=None, # Size of batch for validation `DataLoader`
+        **kwargs
+    ):
```

Comment:
```suggestion
    )->DataLoaders:
```

Reply:
In the [fastai docment Sprint Style Guide](https://github.com/muellerzr/fastai-docment-sprint/blob/master/docs_src/style-guide.md), there seems to be spaces around `->`. May be I can add that in my change?

Reply:
Definitely.  Good call on the style guide.  I couldn't remember for sure